src/providers/strategies/WalletConnectProviderStrategy/types/walletConnect.types.ts

```diff
@@ -22,6 +22,7 @@ export enum WalletConnectEventsEnum {
 
 export interface IWalletConnectModalData {
   wcURI: string;
+  walletConnectDeepLink: string;
```

Comment:
src/managers/internal/WalletConnectStateManager/WalletConnectStateManager.ts same interface

Reply:
Discussed, it's the same interface, in one instance it's declared, in the other it's called.